flowmachine/tests/test_cache.py

```diff
@@ -182,7 +182,7 @@ def test_invalidate_cache_multi(flowmachine_connect):
     assert not cache_table_exists(flowmachine_connect, dl1.md5)
     assert not cache_table_exists(flowmachine_connect, hl1.md5)
     has_deps = bool(flowmachine_connect.fetch("SELECT * FROM cache.dependencies"))
-    assert has_deps
+    assert not has_deps
```

Comment:
This test is duped I think - remove in favour of the one on L166, which correctly inverts the last assert?